client/client.go

```diff
@@ -336,3 +344,52 @@ func (x *PrmDial) setDialFunc(connFunc connFunc) {
 	}
 	x.customConnFunc = connFunc
 }
+
+// NewGRPC constructs Client from the provided gRPC connection with options.
+// Resulting client is ready for RPCs, [Client.Dial] must not be called for it.
+// All requests, except ops accepting signer parameter, are signed with
+// randomized private key in [neofscrypto.ECDSA_DETERMINISTIC_SHA256] scheme.
+//
+// The signBufPool is required and must include objects of *[]byte type.
+//
+// The streamMsgTimeout must not be negative. It defaults to 10s when zero.
+//
+// The respInterceptor is optional.
+//
+// Experimental: the function focuses on SN system needs and is not recommended
+// for use by regular apps. May be removed in future releases.
+func NewGRPC(conn *grpc.ClientConn, signBufPool *sync.Pool, streamMsgTimeout time.Duration, respInterceptor func(pub []byte) error) (*Client, error) {
+	if streamMsgTimeout < 0 {
+		panic(fmt.Sprintf("negative stream message timeout %v", streamMsgTimeout))
+	}
+	if signBufPool == nil {
```

Comment:
Why mandatory?

Reply:
based on the SN use case

Reply:
It's a widely available SDK API, SN can pass it, but others can ignore it if needed. These pools are really unobvious for any external users.

Reply:
It doesn't matter, SDK should be as user-friendly as it can be. It costs nothing to provide the same experience as with regular New here, so why introducing artificial limitations?

Reply:
I wouldn't state any limit artificial w/o real practical problem. Prm can be optionalized at any time, if someone will rly need this

Reply:
Well, it's SDK. We _always_ have to think about the way any of these APIs can be used or misused externally.

Reply:
i understand ur philosophy. The prm is optional now

Reply:
Push?